sdcm/nemesis.py

```diff
@@ -1037,7 +1037,8 @@ def disrupt_network_random_interruptions(self):  # pylint: disable=invalid-name
 
         # get the last 10min avg network bandwidth used, and limit  30% to 70% of it
         prometheus_stats = PrometheusDBStats(host=self.monitoring_set.nodes[0].external_address)
-        query = 'avg(irate(node_network_receive_bytes_total{instance=~"%s", device="eth0"}[30s]))' % self.target_node.ip_address
+        query = 'avg(irate(node_network_receive_bytes_total{instance=~"[%s]", device="eth0"}[30s]))' % \
```

Comment:
I would change to `=~".*?%s.*?"` to support both cases

Reply:
Ok

Reply:
Done